general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -109,6 +109,18 @@ def request() -> requests.Response:
 
         return self._make_request(request).json()
 
+    def get_superstaq_version(self) -> Dict[str, Optional[str]]:
```

Comment:
if #413 was approved you can go ahead an merge so these don't show up here

Reply:
Yeah, I didn't merge that yet because the Notebook check failed. Should I still squash and merge?

Reply:
yeah it should be fine - the pr doesn't do anything that would affect the notebooks, and it looks like one failure is because of #396 and the rest should be fixed by #418